stores/amazon_monitoring.py

```diff
@@ -123,8 +137,7 @@ def __init__(
         delay: float,
         amazon_config,
         tasks=1,
-        checkshipping=False,
-    ) -> None:
+        checkshipping=False,) -> None:
```

Comment:
This repository uses a python formatting tool called black that automatically formats the code. You can run it on your code and it will clean it up (which among other things, will undo this change)

Reply:
yeah once it's ready to get pulled I'll look into running it on the code